### DISCUSSION TITLE
ASSET PREVIEW    works with  excel-a.xlsx    but not with  excel-b.xlsx

### DISCUSSION THREAD

**Discussion by ismetsener** [pimcore/pimcore#9424] [Q&A / Support] [answered]
Hello,

I had some excel files uploaded to the asset folder.
there was a extra tab Preview,  here I could see the content of excel files.

but some days later I added new excel files, now I cannot see the content of the new uploaded excel files. There is  no 
Preview tab
but the strange thing is  I can still see the preview of the excel files I uploaded the first time
what also is strange is , when I make a copy of the first uploaded excel files, I still cannot see the content (no preview tab)

kind regards,
Ismet

**Comment by dvesh3** [comment, marked as answer]
please make sure maintenance job is setup correctly for your application, which generates the preview for documents asynchronously in background https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html#page_5-Maintenance-Cron-Job

**Reply by ismetsener** [reply]
i get error with the maintenance cronjob:


Output:
================
GPL Ghostscript 9.25 (2018-09-13)
Copyright (C) 2018 Artifex Software, Inc.  All rights reserved.
This software comes with NO WARRANTY: see the file PUBLIC for details.
Error: /undefined in PK
Operand stack:

Execution stack:
   %interp_exit   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--   --nostringval--   --nostringval--   false   1   %stopped_push   2047   1   3   %oparray_pop   2046   1   3   %oparray_pop   2026   1   3   %oparray_pop   1884   1   3   %oparray_pop   --nostringval--   %errorexec_pop   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--
Dictionary stack:
   --dict:963/1684(ro)(G)--   --dict:0/20(G)--   --dict:77/200(L)--
Current allocation mode is local
Current file position is 3


Error Output:
================
GPL Ghostscript 9.25: Unrecoverable error, exit code 1
 in /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/Process/Process.php:267
Stack trace:
#0 /opt/pimcore/vendor/pimcore/pimcore/lib/Document/Adapter/Ghostscript.php(258): Symfony\Component\Process\Process->mustRun()
#1 /opt/pimcore/vendor/pimcore/pimcore/lib/Document/Adapter/LibreOffice.php(206): Pimcore\Document\Adapter\Ghostscript->getText()
#2 /opt/pimcore/vendor/pimcore/pimcore/models/Asset/Document.php(125): Pimcore\Document\Adapter\LibreOffice->getText()
#3 /opt/pimcore/vendor/pimcore/pimcore/models/Search/Backend/Data.php(440): Pimcore\Model\Asset\Document->getText()
#4 /opt/pimcore/vendor/pimcore/pimcore/bundles/CoreBundle/EventListener/SearchBackendListener.php(78): Pimcore\Model\Search\Backend\Data->setDataFromElement()
#5 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php(117): Pimcore\Bundle\CoreBundle\EventListener\SearchBackendListener->onPostUpdateElement()
#6 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php(230): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke()
#7 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php(59): Symfony\Component\EventDispatcher\EventDispatcher->callListeners()
#8 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php(151): Symfony\Component\EventDispatcher\EventDispatcher->dispatch()
#9 /opt/pimcore/vendor/pimcore/pimcore/models/Asset.php(601): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch()
#10 /opt/pimcore/vendor/pimcore/pimcore/lib/Maintenance/Tasks/AssetDocumentConversionTask.php(60): Pimcore\Model\Asset->save()
#11 /opt/pimcore/vendor/pimcore/pimcore/lib/Maintenance/Executor.php(97): Pimcore\Maintenance\Tasks\AssetDocumentConversionTask->execute()
#12 /opt/pimcore/vendor/pimcore/pimcore/bundles/CoreBundle/Command/MaintenanceCommand.php(97): Pimcore\Maintenance\Executor->executeMaintenance()
#13 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php(288): Pimcore\Bundle\CoreBundle\Command\MaintenanceCommand->execute()
#14 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php(992): Symfony\Component\Console\Command\Command->run()
#15 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php(96): Symfony\Component\Console\Application->doRunCommand()
#16 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php(291): Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand()
#17 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php(82): Symfony\Component\Console\Application->doRun()
#18 /opt/pimcore/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php(167): Symfony\Bundle\FrameworkBundle\Console\Application->doRun()
#19 /opt/pimcore/bin/console(46): Symfony\Component\Console\Application->run()
#20 {main}

**Reply by dvesh3** [reply]
this is coming from Ghostscript, so try updating it on your server.

**Reply by ismetsener** [reply]
Thanks
Problem solved by running maintenance job 
ghostscript was already installed right

this can be closed
